updating README (#169)
DataDog/cloudformation-template: aws_quickstart/README.md

# Datadog AWS Integration

> [![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=datadog&templateURL=https://datadog-cloudformation-template.s3.amazonaws.com/aws/v2.0.6/main_v2.yaml)

Author: Thanks for the catch, updated.

  
## Installation

1. Open the [AWS integration tile](https://app.datadoghq.com/account/settings#integrations/amazon-web-services) within the Datadog platform.
   1. Click "Add AWS Account(s)".
   1. Fill out the various fields under "Automatically using CloudFormation".
   1. Click "Launch CloudFormation Template" in the bottom right.

> 1. Log into your admin AWS account/role and deploy the CloudFormation Stack that you're linked to. All required paramaters will be filled in.

Reviewer: ```suggestion
1. Log into your admin AWS account/role and deploy the CloudFormation Stack that you're linked to. All required parameters will be filled in.
```

>    1. Optinally edit "Advanced" fields to set the IAM RoleName used, and disable default collection settings.

Reviewer: ```suggestion
   1. Optionally edit "Advanced" fields to set the `IAMRoleName` used, or disable default collection settings.
```
